server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRToAuditList.java

```diff
@@ -152,7 +152,10 @@ public String endpoint(final Request the_request, final Response the_response) {
       int start = index;
       int end = start + ballot_count - 1; // end is inclusive
       
-      while (cvr_set.size() < ballot_count && cvr_set.size() < county_ballots) {
+      // if duplicates is set we go until the list has the right number; if not,
+      // we go until we hit the end of our CVR pool
+      while (duplicates && cvr_to_audit_list.size() < ballot_count || 
```

Comment:
Nit: I'd prefer we don't rely on operator precedence here, and instead explicitly use parens.

Reply:
I would too, generally... but I didn't feel like putting in a `@SuppressWarnings` for it.

Reply:
(I had written it with parens in the first place)